.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+  - 2.6
+  - 2.7
+  - 3.3
+  - 3.4
+  - 3.5
+  - pypy
+  - pypy3
+install:
+- python setup.py develop
+- pip install querystringsafe_base64[tests]
+- pip install coveralls
+script:
+- py.test --verbose --capture=no --showlocals --cov src/querystringsafe_base64
+- pylama
```

Comment:
It's missing `pylama.ini`.

```
[pylama]
linters = pep8,pep257,pyflakes,mccabe

[pylama:pep8]
max_line_length = 120

[pylama:*/__init__.py]
# W0611 - Ignore warnings about missing module-level docstrings in __init__.py files.
# D100 - Ignore 'imported but not used' warnings in __init__.py files.
ignore = W0611,D100
```

You can also add pylint. `pep257` will require you to fix new lines in docstrings. Have some fame :D